support/source/ios/build.sh

```diff
@@ -39,9 +39,9 @@ if [ -d "./build" ]; then
 rm -rf "./build"
 fi
 
-echo "Mockifer: iOS Framework: Build complete! Find '${FRAMEWORK_NAME}.framework' in the 'products/libs/ios' folder. See the 'sample-ios' project to find out how to integrate the framework."
+echo "Mockifer: iOS Framework: Build complete! Find '${FRAMEWORK_NAME}.framework' in the 'Automation/MockiferEcobee/FrameworkBuild' folder. See the 'sample-ios' project to find out how to integrate the framework."
```

Comment:
Hi, can you please revert this file as well it appears to reflect your local file system instead of the products folder.

Reply:
Done